game_controller.py

```diff
@@ -97,10 +99,11 @@ def _set_priorities(self) -> None:
         StealEffect.__priority__ = 3
         SpyEffect.__priority__ = 4
         FirstNightEffect.__priority__ = 5
+        TrapEffect.__priority__ = 6
 
         for citizen, counter in zip(self.citizens_dict,
                                     range(0, len(self.citizens_dict))):
-            citizen["card"].effect.__priority__ = counter + 6
+            citizen["card"].effect.__priority__ = counter + 7
```

Comment:
This code could be rewritten to prevent magic numbers usage:
Smth like this (Use the below code as a reference. Not as an exact solution):

```

standard_effects = [
        NoneEffect,
        StagingEffect,
        KillEffect,
        StealEffect,
        SpyEffect,
        FirstNightEffect,
        TrapEffect
        ]
all_effects = []
all_effects.extend(standard_effects)
all_effects.extend(self.citizens_dict)

for i in range(0, len(all_effects)):
        all_effects[i].effect.__priority__ = i
```

Reply:
Done